llvm/test/tools/llvm-readobj/ELF/note-core-multiple-sections.test

```diff
@@ -0,0 +1,39 @@
+## Test that note values are interpreted correctly for core files with multiple sections.
```

Comment:
What's the relevance of this being a core file to the actual behaviour you want to test?

Reply:
removed